embed-code-go/embedding/parsing/parsing_context.go

```diff
@@ -147,6 +154,16 @@ func (pc *ParsingContext) ResolveEmbeddingNotFound() {
 	pc.EmbeddingsNotFound = append(pc.EmbeddingsNotFound, currentEmbedding.Embedding)
 }
 
+// Deletes embedding from the list of embeddings if it is not accepted.
+//
+// Also appends it to the list of such embeddings for logging.
+func (pc *ParsingContext) ResolveEmbeddingNotAccepted() {
```

Comment:
Lets make it more readable I propose to avoid `EmbedingNotAccepted` and replace it with `UnacceptedEmbedding` or something similar both in method and field names. Now it sounds a little bit Yoda-style.